README.md

```diff
@@ -67,13 +72,66 @@ This will deploy the application to AWS using CDK.
    ```sh
    yarn workspace cdk cdk deploy --all
    ```
+1. View your application resources in CloudFormation. If you go to the stack IotApp -> Outputs you can see the URL that the application will be available from.
 
 To specify a stack name, use the --context (-c) option, as shown in the following example.
 
 ```sh
 yarn workspace cdk cdk deploy -c stackName=my-stack-name
 ```
 
+## AWS IAM Identity Center (SSO) integration:
+If you wish to use IAM Identity Center as your authentication source, there are some manual steps to complete first.
+
+This integration is based on [these integration instructions](https://repost.aws/knowledge-center/cognito-user-pool-iam-integration).
+
+#### Gather information from Cloudformation
+Go to the CloudFormation console to get required resource information. You will need to take note of the following values: `domain-prefix`, `userpool-id`, `client-id` and `signin-url`. These will be used to configure IAM Identity Center as an authentication source.
+1. Find the stack called `IotApp`
+   * Go to the outputs tab and note down the value for `AppUrl` as the `signin-url` that will be used later.
+1. Find the stack called `IotAppAuth`
+   * Go to the resources tab and expand the Auth section.
+   * Note down the physical id value for `UserPoolDomain` as `domain-prefix`
+   * Note down the physical id value for `UserPool` as `userpool-id`
+   * Note down the physical id value for `UserPoolClient` as `client-id`
+#### Configure IAM Identity Center
```

Comment:
this is good for setting up 👍
Unfortunate SSO doesn't support doing it programmatically today; hopefully SSO can support this use case in the future.